libres/lib/enkf/enkf_config_node.cpp

```diff
@@ -43,1002 +42,1082 @@
-  return config_node->impl_type;
+ert_impl_type
+enkf_config_node_get_impl_type(const enkf_config_node_type *config_node) {
+    return config_node->impl_type;
 }
 
-
-enkf_var_type enkf_config_node_get_var_type(const enkf_config_node_type *config_node) {
-  return config_node->var_type;
+enkf_var_type
+enkf_config_node_get_var_type(const enkf_config_node_type *config_node) {
+    return config_node->var_type;
 }
 
-
-const char * enkf_config_node_get_key(const enkf_config_node_type * config_node) { return config_node->key; }
-
-
-const stringlist_type  * enkf_config_node_get_obs_keys(const enkf_config_node_type *config_node) {
-  return config_node->obs_keys;
+const char *enkf_config_node_get_key(const enkf_config_node_type *config_node) {
+    return config_node->key;
 }
 
-
-int enkf_config_node_get_num_obs( const enkf_config_node_type * config_node ) {
-  return stringlist_get_size( config_node->obs_keys );
+const stringlist_type *
+enkf_config_node_get_obs_keys(const enkf_config_node_type *config_node) {
+    return config_node->obs_keys;
 }
 
+int enkf_config_node_get_num_obs(const enkf_config_node_type *config_node) {
+    return stringlist_get_size(config_node->obs_keys);
+}
 
 /**
    This checks the index_key - and sums up over all the time points of the observation.
 */
 
-int enkf_config_node_load_obs( const enkf_config_node_type * config_node , enkf_obs_type * enkf_obs ,const char * key_index , int obs_count , time_t * _sim_time , double * _y , double * _std) {
-  ert_impl_type impl_type = enkf_config_node_get_impl_type(config_node);
-  int num_obs = 0;
-  int iobs;
-
-  for (iobs = 0; iobs < stringlist_get_size( config_node->obs_keys ); iobs++) {
-    obs_vector_type * obs_vector = enkf_obs_get_vector( enkf_obs , stringlist_iget( config_node->obs_keys , iobs));
-
-    int report_step = -1;
-    while (true) {
-      report_step = obs_vector_get_next_active_step( obs_vector , report_step);
-      if (report_step == -1) break;
-      {
-        bool valid;
-        double value , std1;
-
-        /**
+int enkf_config_node_load_obs(const enkf_config_node_type *config_node,
+                              enkf_obs_type *enkf_obs, const char *key_index,
+                              int obs_count, time_t *_sim_time, double *_y,
+                              double *_std) {
+    ert_impl_type impl_type = enkf_config_node_get_impl_type(config_node);
+    int num_obs = 0;
+    int iobs;
+
+    for (iobs = 0; iobs < stringlist_get_size(config_node->obs_keys); iobs++) {
+        obs_vector_type *obs_vector = enkf_obs_get_vector(
+            enkf_obs, stringlist_iget(config_node->obs_keys, iobs));
+
+        int report_step = -1;
+        while (true) {
+            report_step =
+                obs_vector_get_next_active_step(obs_vector, report_step);
+            if (report_step == -1)
+                break;
+            {
+                bool valid;
+                double value, std1;
+
+                /**
```

Comment:
Indentation of comment now looks weird because the indentation of the code has changed.